dvc/utils/__init__.py

```diff
@@ -512,3 +513,11 @@ def errored_revisions(rev_data: Dict) -> List:
         if nested_contains(data, "error"):
             result.append(revision)
     return result
+
+
+def getenv_bool(name: str, default: str = "False"):
+    raw = os.getenv(name, default).title()
+    try:
+        return bool(ast.literal_eval(raw))
+    except ValueError:
+        return False
```

Comment:
I don't think we should get into this kind behavior, in my opinion it would be better to just document that if the environment variable is set at all, it will be considered enabled.

but others may have different thoughts on this
cc: @iterative/dvc

Reply:
I could see a scenario where a templated YAML workflow yields `env: VAR: false` but not `env: VAR: False`. Agree that it seems to much of a headache to support.

`if os.getenv(name, '')` seems fine to me...

Reply:
We also have `env2bool` in dvc/utils, that we can use here instead.